tests/spread/rockcraft/plugin-poetry/task.yaml

```diff
@@ -0,0 +1,26 @@
+summary: Poetry plugin test
+
+execute: |
+  ROCK_FILE="plugin-poetry_0.1_amd64.rock"
+  IMAGE="plugin-poetry:0.1"
+  
+  # Build the rock & load it into docker
+  run_rockcraft pack
+  test -f ${ROCK_FILE}
+  sudo rockcraft.skopeo --insecure-policy copy oci-archive:${ROCK_FILE} docker-daemon:${IMAGE}
+  docker images
+  rm ${ROCK_FILE}
+  
+  # Run the packaged project, both via the console script and via "python -m"
+  docker run --rm $IMAGE exec test-poetry | MATCH "Hello from Poetry"
+  docker run --rm $IMAGE exec /bin/python3 -m test_poetry | MATCH "Hello from Poetry"
+  docker run --rm $IMAGE exec /usr/bin/python3 -m test_poetry | MATCH "Hello from Poetry"
+  docker run --rm $IMAGE exec python3 -m test_poetry | MATCH "Hello from Poetry"
+  
+  # Run the extra Python package, installed as a dependency, to make sure it's found
+  docker run --rm $IMAGE exec black --version
+  docker run --rm $IMAGE exec /bin/python3 -m black --version
+  docker run --rm $IMAGE exec /usr/bin/python3 -m black --version
+  docker run --rm $IMAGE exec python3 -m black --version
+
+  docker system prune -a -f
```

Comment:
Doesn't this really belong in the `restore` part (in case one of the Docker commands fails)?

Or perhaps even in the `spread.yaml`'s `restore-each`?